README.md

```diff
@@ -138,27 +142,22 @@ But first we'll need to get a local version of Optimistic Ethereum node running.
 
 ---
 
-Fortunately, we have a [handy dandy repository](https://github.com/ethereum-optimism/optimism-integration) that makes it easy to spin up a local Optimistic Ethereum node!
+Fortunately, we have a [handy dandy monorepo](https://github.com/ethereum-optimism/optimism) that makes it easy to spin up a local Optimistic Ethereum node!
 
 Since we're going to be using Docker, make sure that Docker is installed on your machine prior to moving on (info on how to do that [here](https://docs.docker.com/engine/install/)).
 
 Now we just need to install our Optimistic Ethereum node by running:
 
 ```sh
-git clone git@github.com:ethereum-optimism/optimism-integration.git --recurse-submodules
-cd optimism-integration
-./pull.sh
+git clone git@github.com:ethereum-optimism/optimism.git
+cd optimism
+yarn
+yarn build
+docker-compose build
+docker-compose up
 ```
 
-`./pull.sh` will pull the latest version of all of our docker images and make sure everything else is up to date.
-
-Then we'll run the  `./up.sh` command to spin up our node:
-
-```sh
-./up.sh
-```
-
-Give `./up.sh` a little bit to fully start up (could be up to ~30 seconds).
+Give the `docker-compose up -d` command a little bit to fully start up (could be up to ~30 seconds).
```

Comment:
NIT, but also a general comment about the approach to writing docs: 
IMO the language should be less informal and more direct/concise. People are mostly just trying to get the info they need. Although the phrasing here feels very straightforward to native english speaker, it may not be to others. Even for native speakers, it still requires more careful reading to extract the information they've come here for.